sweep/sweeper.go

```diff
@@ -1019,7 +1020,7 @@ func (s *UtxoSweeper) scheduleSweep(currentHeight int32) error {
 
 	// Start sweep timer to create opportunity for more inputs to be added
 	// before a tx is constructed.
-	s.timer = s.cfg.NewBatchTimer()
+	s.timer = time.NewTicker(s.cfg.TickerDuration).C
```

Comment:
Stop the prior ticker before? Would require changing the variable here or adding another one to be able to capture more than just the notification channel.

Reply:
Could do. Since this is removed in the next commit, so I guess it's fine to leave it as-is?